### PR TITLE
Use bignum static constants instead of BN_bin2bn()

### PR DESCRIPTION
**Pull request by huangqinjin**
keep consistent with bn_dh.c bn_nist.c bn_srp.c

### CODE REVIEW

**Comment by richsalz** [comment]
rather than  cut/paste and duplicate the bn_pack macros, they should be put into one place in bn_lcl.h and used everywhere consistently.

File: crypto/bn/bn_const.c [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #include <openssl/bn.h>
 
+#if 0
```
Comment (mattcaswell) [inline crypto/bn/bn_const.c]: Why `#if 0` the old code out rather than just deleting it?

**Comment by huangqinjin** [comment]
@richsalz ` bn_srp.c` defined the `bn_pack4` macro, but I think `bn_pack2` marco should work well. How to make them consistent and then put into `bn_lcl.h`.This is my first try to contribute to openssl, so any word helps me. Thank you.

**Comment by richsalz** [comment]
Post to openssl-dev; good questions.

**Pull request closed without merging**